prow/plank/controller_test.go

```diff
@@ -1588,7 +1588,13 @@ func TestSyncPendingJob(t *testing.T) {
 			if actual := actual.Complete(); actual != tc.ExpectedComplete {
 				t.Errorf("expected complete: %t, got complete: %t", tc.ExpectedComplete, actual)
 			}
-
+			maxPodUnscheduled := r.config().Plank.PodUnscheduledTimeout.Duration 
```

Comment:
This doesn't exercise the code path that you added in `reconciler.go`. 

Please read https://dave.cheney.net/2013/06/09/writing-table-driven-tests-in-go, try to understand how this test function works, then add at least test case under the table